server/src/integration/java/org/opentosca/toscana/plugins/kubernetes/docker/image/PushingImageBuilderIT.java

```diff
@@ -0,0 +1,103 @@
+package org.opentosca.toscana.plugins.kubernetes.docker.image;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import org.opentosca.toscana.core.transformation.TransformationContext;
+import org.opentosca.toscana.plugins.kubernetes.docker.DockerTestUtils;
+import org.opentosca.toscana.plugins.kubernetes.docker.util.DockerRegistryCredentials;
+
+import com.spotify.docker.client.DefaultDockerClient;
+import com.spotify.docker.client.DockerClient;
+import com.spotify.docker.client.messages.ContainerConfig;
+import com.spotify.docker.client.messages.HostConfig;
+import com.spotify.docker.client.messages.PortBinding;
+import org.junit.After;
+import org.junit.Assume;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static java.util.Collections.singletonMap;
+
+public class PushingImageBuilderIT extends ExportingImageBuilderIT {
+
+    private static Logger logger = LoggerFactory.getLogger(PushingImageBuilderIT.class);
+
+    private DockerRegistryCredentials creds;
+
+    private String tag = "sha-test-" + Long.toString(System.currentTimeMillis(), 16);
+
+    private String registryid;
+
+    @Override
+    public void init() throws Exception {
+//        Assume.assumeNotNull(
```

Comment:
Please delete this if this is no needed?

Reply:
I would keep it in, because that is the part used to run the test against a different registry.